src/cli-command.ts

```diff
@@ -236,8 +236,8 @@ export class Command {
     return newTknCommand('taskrun', 'logs', name, '-f');
   }
 
-  static checkTekton(): CliCommand {
-    return newK8sCommand('auth', 'can-i', 'create', 'pipeline.tekton.dev', '&&', 'kubectl', 'get', 'pipeline.tekton.dev');
+  static checkUserAuthentication(): CliCommand {
```

Comment:
In this patch i don't do any distinction, as for me this check does not make much sense and it's time consuming.
This call is made at tree creation so what's the point of knowing if the user can create a resource? It should be made when someone wants to create a resource. Then if we want to be picky it lacks the check for the delete as well. 
In my mind this should be done in a different time or at least in a different way. If I'm allowed to get the resources I should be able to see the tree, no?

Reply:
I agree with you but if you want to check if user is connected to a cluster, you should rather issue `kubectl get pods` rather than kubectl `get pipeline.tekton.dev` as the later may return an error is Tekton is not installed